Stripe/PublicHeaders/STPCard.h

```diff
@@ -110,6 +110,11 @@ typedef NS_ENUM(NSInteger, STPCardFundingType) {
 /**
  *  The cardholder's address.
  */
+@property(nonatomic, copy, nullable) STPAddress *address;
```

Comment:
Ah i see, didn't realize there was a different implementation in `STPCard`. Looks fine then!